### ISSUE TITLE
Export animated model as obj

### ISSUE THREAD

**Issue by nitish11** [andrewfengusa/SMPLX_Visualizer#1]
First of all, thank you for the awesome project.

Could you please explain how can we export "Animated 3D model" as .fbx or .obj using python,  instead of writing it to video file as in below line : https://github.com/andrewfengusa/SMPLX_Visualizer/blob/main/SMPLX_mesh_render.py#L105

**Comment by andrewfengusa** [comment]
Yes, you could use the trimesh object: https://github.com/andrewfengusa/SMPLX_Visualizer/blob/805f06b37d09f4373cb8b86b83599185eaf150f1/SMPLX_mesh_render.py#L62

Instead of writing the video frames, you could do recon_body_mesh.export to write out the mesh for each individual frames.

**Comment by nitish11** [comment]
@andrewfengusa  : thank you for your instructions. I am able to able to export 3D mesh model for each frame. Could you please suggest how can I create 3D model with animation from multiple mesh models?
